overlay/update_domains.yml

```diff
@@ -0,0 +1,236 @@
+---
```

Comment:
Is this not redundant to the [overlay/change_deployment_and_network.yml](https://github.com/genesis-community/cf-app-autoscaler-genesis-kit/pull/28/files#diff-c27101fc608d54840435845ab00a729815cd324b01c83bc6ee62b655946ec21d) ?

This will not merge correctly because of the bosh variable ((deployment_name))

Reply:
domain gets updated on overlay/update_domains.yml and deployment/network on overlay/change_deployment_and_network.yml using that deployment name. The bosh variable has been added on overlay/base.yml

Reply:
Merge order from base manifest should resolve this, if not we'll circle back with a point release.